DirectProgramming/DPC++FPGA/Tutorials/Tools/experimental/platform-designer/add-quartus/system_console/load_vals_host-pipes.tcl

```diff
@@ -0,0 +1,13 @@
+#  Copyright (c) 2022 Intel Corporation                                  
```

Comment:
Same remark: I hope that we can avoid shipping this file and all the tcl handwritten files, or nicely explain what they are and why we shipped them with the sample.
No one should even try to look at these

Reply:
see comment for jtag_avmm.tcl